code/modules/VR/vr_sleeper.dm

```diff
@@ -68,10 +72,10 @@
 			SStgui.close_user_uis(occupant, src)
 		..()
 
-/obj/machinery/vr_sleeper/close_machine()
-	..()
-	if(occupant)
-		ui_interact(occupant)
+/obj/machinery/vr_sleeper/MouseDrop_T(mob/target, mob/user)
+	if(user.stat || user.lying || !Adjacent(user) || !user.Adjacent(target) || !iscarbon(target) || !user.IsAdvancedToolUser())
```

Comment:
you might benefit from copying this from the cryo tub if getting put inside a sleeper has any annoying side effects

Reply:
It doesn't.